src/match.cpp

```diff
@@ -220,14 +224,12 @@ std::vector<MatchResult> regexbench::match(Engine& engine,
                                   (pLogger ? pLogger.get() : nullptr)));
   }
 
-  uint32_t sec = 0;
-  bool realTime = true;
+  gettimeofday(&begin, NULL);
```

Comment:
Why don't we add say `begintime` in `MatchResult` struct and use it instead of using local variable `begin` and change signature of the function `statistic`?